framework/include/media/InputDataSource.h

```diff
@@ -68,6 +68,13 @@ class InputDataSource : public DataSource
 	 */
 	virtual size_t read(unsigned char* buf, size_t size) = 0;
 
+	/**
+	 * @brief Sets the position of stream
+	 * @details @b #include <media/InputDataSource.h>
+	 * @since TizenRT v2.0
+	 */
+	virtual int seek(long offset, int origin) = 0;
```

Comment:
Actually inputDataSource is used for reading from stream buffer,
Let's apply polymorphism here.
```
virtual size_t read(unsigend char* buf, size_t size);

virtual size_t read(unsigend char* buf, off_t offset, size_t size);
```
Instead of seek()->read(), we can call just read().